examples/torch/object_detection/README.md

```diff
@@ -11,6 +11,12 @@ This sample demonstrates DL model compression capabailites for object detection
 
 ## Installation
 
+At this point it is assumed that you have already installed nncf
+
+```
+pip install nncf
```

Comment:
You are only referencing the PyPI path. Should also reference all the other ways to install NNCF if you are mentioning this; also, should update all other READMEs for the examples.

Reply:
done